apps/wallet/src/ui/app/components/WalletListSelectItem.tsx

```diff
@@ -76,30 +81,43 @@ export function WalletListSelectItem({
             clearTimeout(timeout);
         };
     }, [isNew]);
+
+    const isActiveModeDisconnect = mode === SELECTION_MODE.DISCONNECT;
+    const isActiveModeSelect = mode === SELECTION_MODE.SELECT;
+
     return (
         <div ref={elementRef} className={styles({ selected, mode, disabled })}>
-            {mode === 'select' ? (
+            {isActiveModeSelect ? (
                 <CheckFill16
                     className={cx(
                         selected ? 'text-success' : 'text-gray-50',
                         'transition text-base font-bold'
                     )}
                 />
             ) : null}
-            {mode === 'disconnect' && selected ? (
+            {isActiveModeDisconnect && selected ? (
                 <XFill16 className="text-issue-dark text-base font-bold" />
             ) : null}
-            <Text mono variant="body" weight="semibold">
+            <Text
+                mono
+                variant="body"
+                weight="semibold"
+                color={
+                    selected && isActiveModeDisconnect
```

Comment:
I think in line 33 `compoundVariants` we do most of the handling for selecting the color of the component. Why not update that?

Reply:
@pchrysochoidis updated it to follow that convention! Didn't realize that's what the cva stuff was for beforehand 👍